<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stu_check_score</class>
 <widget class="QWidget" name="stu_check_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <widget class="QWidget" name="all_score_tab">
      <attribute name="title">
       <string>所有科目成绩</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="check_all_title">
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>查询成绩</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>88</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="check_score_all_title">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>该学生的成绩为：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>148</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="styleSheet">
              <string notr="true">
color: rgb(0, 0,0);</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="1" colspan="2">
               <widget class="QLineEdit" name="check_physics_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="check_math_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>数学：</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QLineEdit" name="check_sum_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="check_Chinese_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;语文：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="check_chemistry_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>化学：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="check_English_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>英语：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="check_physics_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>物理：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QLineEdit" name="check_English_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QLineEdit" name="check_math_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="check_biology_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>生物：</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="2">
               <widget class="QLineEdit" name="check_chemistry_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="2">
               <widget class="QLineEdit" name="check_biology_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QLineEdit" name="check_Chinese_line">
                <property name="styleSheet">
                 <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QLabel" name="check_sum_label">
                <property name="styleSheet">
                 <string notr="true">font: 16pt &quot;宋体&quot;;</string>
                </property>
                <property name="text">
                 <string>总分：</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>528</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_check_all_return">
             <property name="styleSheet">
              <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(204, 204, 191);</string>
             </property>
             <property name="text">
              <string>返回</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="single_score_tab">
      <attribute name="title">
       <string>单科成绩</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="check_subject">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(243, 243, 255);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 10pt &quot;宋体&quot;;
color: rgb(103, 103, 103);</string>
             </property>
             <property name="placeholderText">
              <string>Places input subject</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="check_score_single_title">
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>该科目的成绩为：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="check_subject_score">
             <property name="styleSheet">
              <string notr="true">border:1px groove gray;border-radius:
7px;padding:2px 4px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_check_single_return">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(204, 204, 191);</string>
           </property>
           <property name="text">
            <string>返回</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>48</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_check_single_confirm">
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;宋体&quot;;
background-color: rgb(218, 205, 255);</string>
           </property>
           <property name="text">
            <string>确认查询</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>138</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
